jaeger-core/src/main/java/com/uber/jaeger/Span.java

```diff
@@ -236,30 +236,21 @@ public Span log(long timestampMicroseconds, Map<String, ?> fields) {
 
   @Override
   public Span log(String event) {
-    return log(tracer.clock().currentTimeMicros(), event, null);
+    return log(tracer.clock().currentTimeMicros(), event);
   }
 
   @Override
   public Span log(long timestampMicroseconds, String event) {
-    return log(timestampMicroseconds, event, null);
-  }
-
-  @Override
-  public Span log(String message, /* @Nullable */ Object payload) {
```

Comment:
I didn't realize this is being removed in OT - we should only be deprecating in OT

Reply:
Should we raise this, and get a RC2 produced before doing a release?

Reply:
Yes, I think we should - it's a bad practice to just remove a method

Reply:
https://github.com/opentracing/opentracing-java/issues/189#issuecomment-333241372